common/src/main/aidl/com/microsoft/identity/client/IMicrosoftAuthService.aidl

```diff
@@ -32,8 +32,12 @@ interface IMicrosoftAuthService {
 
     Bundle hello(in Bundle bundle);
 
+    Bundle getAccountMode();
+
     Bundle getAccounts(in Bundle bundle);
 
+    Bundle getCurrentAccount();
```

Comment:
1. Should the getCurrentAccount() only be called when the account mode is "single"?
2.  If not, what should it return when multiple accounts exist? 
3. What kind of accounts are returning here? I assume WPJ account only, right?

Reply:
1. Yes
2. An error bundle.
3. an account that holds BRT. In the shared device mode, there will be only 0 or 1 account that holds BRT, and that account MUST not be the WPJ account. (The WPJ account is used by the admin to bootstrap the WPJ/shared device state only)